frontend/src/services/Fetch.jsx

```diff
@@ -1,11 +1,13 @@
 import axios from "axios";
 
-function fetchData(data) {
+function fetchData(lat, long) {
   return axios
     .get(
-      `https://api.open-meteo.com/v1/forecast?latitude=${data.lat}&longitude=${data.long}&daily=weathercode&timezone=Europe%2FBerlin`
+      `https://api.open-meteo.com/v1/forecast?latitude=${lat}&longitude=${long}&daily=weathercode,temperature_2m_max,temperature_2m_min,sunrise,sunset,precipitation_sum,rain_sum,showers_sum,snowfall_sum,windspeed_10m_max,winddirection_10m_dominant&timezone=Europe%2FBerlin`
```

Comment:
![image](https://user-images.githubusercontent.com/23694617/199927840-1a19690d-78f9-4fc6-8323-eae1a80d3ca6.png)
Use `params` option to be clearer
https://masteringjs.io/tutorials/axios/get-query-params